src/components/GlobalLoading/GlobalLoading.tsx

```diff
@@ -1,90 +1,64 @@
 import { FunctionComponent as FC, ReactNode, useEffect, useContext } from 'react';
+import { useStaticQuery, graphql } from 'gatsby';
+
 // Session-related scripts
 import '@ably/ui/core/scripts';
 import { loadSprites } from '@ably/ui/core/scripts';
 import sprites from '@ably/ui/core/sprites.svg';
-import { Script } from 'gatsby';
-import posthog from 'posthog-js';
-import { loadBoomerang } from 'src/third-party/boomerang';
-import { loadHeadway } from 'src/third-party/headway';
 import UserContext from '../../contexts/user-context';
-import { sessionTracking } from './session-tracking';
-
-const hubspotTrackingId = process.env.GATSBY_HUBSPOT_TRACKING_ID;
-const boomerangEnabled = process.env.GATSBY_BOOMERANG_ENABLED;
-const posthogApiKey = process.env.GATSBY_POSTHOG_API_KEY;
-const headwayAccountId = process.env.GATSBY_HEADWAY_ACCOUNT_ID;
-const googleTagManagerAuthToken = process.env.GATSBY_GOOGLE_TAG_MANAGER_AUTH_TOKEN;
-const googleTagManagerPreview = process.env.GATSBY_GOOGLE_TAG_MANAGER_PREVIEW;
 
-const gtmSnippet = `(function (w, d, s, l, i) {
-  w[l] = w[l] || [];
-  w[l].push({ 'gtm.start': new Date().getTime(), event: 'gtm.js' });
-  const f = d.getElementsByTagName(s)[0],
-    j = d.createElement(s),
-    dl = l != 'dataLayer' ? '&l=' + l : '';
-  j.async = true;
-  j.src =
-    'https://www.googletagmanager.com/gtm.js?id=' +
-    i +
-    dl +
-    '&gtm_auth=${googleTagManagerAuthToken}&gtm_preview=${googleTagManagerPreview}&gtm_cookies_win=x';
-  f.parentNode.insertBefore(j, f);
-})(window, document, 'script', 'dataLayer', 'GTM-TZ37KKW');`;
+import externalScriptInjector from 'external-scripts';
```

Comment:
Do you mean `../../external-scripts`?

Reply:
Changed to `src/external-scripts`, must have just rushed o_O